backend/plugin/parser/pg/system_objects.go

```diff
@@ -752,6 +752,16 @@ var (
 		"_timescaledb_internal":    true,
 		"_timescaledb_config":      true,
 		"squeeze":                  true,
+		// citus extension
+		"citus":          true,
```

Comment:
Not related to this PR. I'm worried about someone creating a schema name citus or squeeze e.g. anyway, future improvement.